function-images/tests/producer-consumer/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/MBaczun/producer-consumer
+
```

Comment:
to find the protobuf generated files locally, use 
```replace tests/chained-functions-serving/proto => ./proto```
otherwise Go would be pulling it from your fork